### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by Meriem-ks** [hbldh/bleak#845] [Q&A]
hello , i'm trying to connect python program to send the result of machine learning model via BLE to a mobile application (flutter ,android studio) , I'm using bleak but I don't have idea how to send an receive data , can you help me please ???

**Comment by koenvervloesem** [comment]
It seems you want your mobile app to be able to connect to your machine running a Python program and read its characteristics using BLE, so that makes your program a GATT server. Bleak isn't designed to do that, but you can try [Bless](https://github.com/kevincar/bless/).

**Reply by Meriem-ks** [reply]
how can I connect this code with my application flutter ! if you have more information about Bless can u share it with me please

**Reply by koenvervloesem** [reply]
Maybe start a [discussion in the Bless repository](https://github.com/kevincar/bless/discussions) with your questions.
